CHANGELOG.md

```diff
@@ -9,11 +9,35 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
--
+- The Compliance Operator is now aware of other Kubernetes platforms outside of
+  OpenShift to accomodate running the operator on other platforms. This allows
+  `ScanSettings` to have different behaviors depending on the platform.
+  `ScanSettings` will automatically schedule to `worker` and `master` nodes
+  when running in an OpenShift cluster, maintaining the previous default
+  behavior for OpenShift clusters. When running on AWS EKS, `ScanSettings` will
+  schedule to all available nodes, by default. The operator will also inherit
+  the `nodeSelector` and `tolerations` from the Compliance Operator pod when
+  running on EKS.
+- Introduced a process for writing release notes
 
 ### Fixes
 
--
+- Improved rule parsing to check for extended OVAL definitions ([bug](https://bugzilla.redhat.com/show_bug.cgi?id=2040282))
+  - Previous versions of the operator wouldn't process extended OVAL
+    defintions, allowing some rules to have `CheckType=None`. This version
+    includes support for processing extended defintions so `CheckType` is
+    properly set.
+- Properly detect `MachineConfig` objects used by the Compliance Operator for
```

Comment:
I think the release note looks good! Thanks for writing it up.

I think the second fix about MachineConfig needs some changes. Please let me know if the following sounds good to you:

Previous Compliance Operator assumed the OCP cluster would only have KubeletConfig Machine Config created by KubeletConfig Object. Therefore when a cluster has a custom KubeletConfig Machine Config not generated from KubeletConfig Object, the KubeletConfig remediation will stale in a pending state because it tries to find the owner of that MC, which does not exist. This fix will detect such custom pre-existing KubeletConfig Machine Config, and as a result, KubeletConfig remediations now work as expected.

Reply:
Excellent - I've worked this into the next version. Thanks!